html/pages/addsrv.inc.php

```diff
@@ -18,10 +18,11 @@
         }
     }
 
-    if ($handle = opendir($config['install_dir'].'/includes/services/')) {
+    if ($handle = opendir($config['nagios_plugins'])) {
         while (false !== ($file = readdir($handle))) {
-            if ($file != '.' && $file != '..' && !strstr($file, '.')) {
-                $servicesform .= "<option value='$file'>$file</option>";
+            if ($file != '.' && $file != '..' && !strstr($file, '.') && strstr($file, 'check_')) {
+                list(,$check_name) = explode('_',$file);
```

Comment:
Fix `list(,$check_name) = explode('_',$file);` to `list(,$check_name) = explode('_',$file,2);`. Avoids cutoff where the plugin is called `check_rta_multi` etc.